browser/scripts/player.js

```diff
@@ -247,6 +247,10 @@ function CreatePlayer(cb) {
 	E2.core = new Core()
 	
 	E2.dom.webgl_canvas = $('#webgl-canvas')
+	if (E2.dom.webgl_canvas.length < 1)
```

Comment:
yeah, that makes sense. let's leave it in. if the page doesn't declare the element, it doesn't want to play Vizor content.